scripts/release_scripts/release_info.py

```diff
@@ -108,7 +107,11 @@ def gather_logs(start, stop='HEAD'):
     """
     get_logs_cmd = GIT_CMD_GET_LOGS_FORMAT_STRING.format(
         GROUP_SEP, start, stop)
-    out = common.run_cmd(get_logs_cmd.split(' ')).split('\x00')
+    # The unicode conversion is required because there can be non-ascii
```

Comment:
There should be some tests for this (i.e. that involve unicode chars).

Reply:
Added.